README.md

```diff
@@ -172,6 +172,20 @@ AWS_ACCESS_KEY_ID=
 AWS_SECRET_ACCESS_KEY=
 ```
 
+#### Exasol
+
+Run Exasol on your local machine in docker (for this case .env is preconfigured)
+```
+docker-compose up -d exasol
+```
+
+Run Exasol server somewhere else and set up env variables:
+```bash
+EXASOL_HOST=
```

Comment:
vyplnil bych tam rovnou defaultni hodnoty s komentarem nahore. Prvne momentalne nejsou zrejme z readme a druhak muze to byt takto pripraveno na copy&paste

Reply:
for this case .env is preconfigured co na tom není jasné?

Reply:
nejak mi ta zavorka unikla, omlouvam se